share/chruby/auto.sh

```diff
@@ -22,12 +23,13 @@ function chruby_auto() {
 		chruby_reset
 		unset RUBY_AUTO_VERSION
 	fi
+	: "$old_" # restore $_ (last argument of last command executed)
 }
 
 if [[ -n "$ZSH_VERSION" ]]; then
 	if [[ ! "$preexec_functions" == *chruby_auto* ]]; then
 		preexec_functions+=("chruby_auto")
```

Comment:
I think it may be time to create a special `chruby_auto_bash` function to wrap the Bash-specific logic.